velox/type/tests/TypeUtilTest.cpp

```diff
@@ -35,5 +35,31 @@ TEST(TypeUtilTest, ConcatRowTypes) {
   EXPECT_EQ(type->toString(), expected->toString());
 }
 
+TEST(TypeUtilTest, tryGetHomogeneousRowChild_NonRow) {
```

Comment:
Do not use underscores in test method names.

It seems better to combine all these test cases into a single test.